server/server.js

```diff
@@ -36,7 +40,8 @@ app.post('/writeReview/:productid', (req, res) => {
     if (err) {
       res.status(400).send();
     } else {
-      res.send('new review added');
+      res.setHeader('content-type', 'application/json');
```

Comment:
I think we use nouns instead of verbs for api endpoints for rest standard. 
from https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/#h-use-nouns-instead-of-verbs-in-endpoint-paths
"With the two principles we discussed above in mind, we should create routes like GET /articles/ for getting news articles. Likewise, POST /articles/ is for adding a new article , PUT /articles/:id is for updating the article with the given id. DELETE /articles/:id is for deleting an existing article with the given ID"

Reply:
Good catch! Didn't know that. Thanks a ton. Fixed.